webapp/src/app/components/Demo/PropsTable/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC, useState } from 'react';
+
+import { PropsTableBody } from './PropsTableBody';
+import { PropsTableHead, PropsType as TableHeadPropsType } from './PropsTableHead';
+import { TableWrapper } from './style/TableWrapper';
+import { DEFAULT_COLUMN_WIDTH } from './style/Header';
+
+export interface IColumnProps {
+  /** Название столбца */
+  name: string;
+  /** Заголовок столбца */
+  title: string;
+  /** Ширина столбца. По умолчанию 100px */
+  width?: number;
+  /** Включение многострочности столбца */
+  isMultiLine?: boolean;
+}
+
+export interface IRowProps {
+  /** Уникальный идентификатор строки */
+  id: string;
+  [propName: string]: any;
+}
+
+export type PropsType = {
+  /** Массив столбцов */
+  columns: IColumnProps[];
+  /** Массив строк */
+  rows: IRowProps[];
+  /** Имя класса для переопределения стилей */
+  className?: string;
+  /** data-test-id атрибут для тестирования компонента */
+  dataTestId?: string;
+};
+
+export const PropsTable: FC<PropsType> = (props) => {
+  const { rows, columns, className, dataTestId } = props;
```

Comment:
```suggestion
export const PropsTable: FC<PropsType> = ({ rows, columns, className, dataTestId }) => {
```
Вроде сам props никуда дальше не прокидывается

Reply:
аналогично поправлю в рамках другого пул реквеста